data/mods/Magiclysm/achievements/spells.json

```diff
@@ -0,0 +1,84 @@
+[
+  {
+    "id": "achievement_learn_any_spell",
+    "type": "achievement",
+    "name": "Would-be Wizard",
+    "requirements": [ { "event_statistic": "num_spells_learned", "is": ">=", "target": 1 } ]
+  },
+  {
+    "id": "achievement_learn_any_spell_3",
```

Comment:
I suggest making each of these "learn n spells" achievements be hidden by the previous one.